tests/functional/features/report_only_edited.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Edited report only submission
+  Partners, redhat and community users attempt to publish their chart by submitting
+  report that was edited afyter it was generated by chart-verifier.
+
+  Examples:
+  | vendor_type  | vendor    | report_path               |
+  | partners     | hashicorp | tests/data/report.yaml    |
+  | redhat       | redhat    | tests/data/report.yaml    |
+  | community    | redhat    | tests/data/report.yaml    |
+
+  Scenario Outline: A partner or redhat associate submits an edited report
+    Given the vendor <vendor> has a valid identity as <vendor_type>
+    And a <report_path> is provided
+    And the report includes <tested> and <supported> OpenshiftVersion values and chart <kubeversion> value
+    When the user sends a pull request with the report
+    Then the pull request is not merged
+    And user gets the <message> in the pull request comment
+
+    Examples:
+      | tested | supported | kubeversion | message                                   |
+      | 4.9    | 4.6-4.9   | >=1.20.0    | is not a valid semantic version           |
```

Comment:
I thought we handle a semver range like `4.6-4.9`, which part of this case is an invalid semver?

Reply:
```4.6 - 4.9```  is a valid semver range - it requires the spaces.